scripts/distribution/ubuntu-packages/template/debian/concordium-node.service

```diff
@@ -45,10 +45,6 @@ Environment=CONCORDIUM_NODE_CONNECTION_BOOTSTRAP_NODES=${build_bootstrap}
 Environment=CONCORDIUM_NODE_CONNECTION_DESIRED_NODES=5
 # maximum number of __nodes__ the node will be connected to.
 Environment=CONCORDIUM_NODE_CONNECTION_MAX_ALLOWED_NODES=10
-# address of the GRPC server
-Environment=CONCORDIUM_NODE_RPC_SERVER_ADDR=0.0.0.0
```

Comment:
So this is what we want to remove in general. We are trying to move away from this API, and we want to make this environment variable conditional. If we do not specify it, will still be present from here. Even worse the build rpc_server:port will be null. But because we overwrite from within the ansible script it has so far been working.

Reply:
What is this comment about? THis has been removed some time ago.

Reply:
Yes. This PR now only removes the map